packages/flutter/lib/src/foundation/assertions.dart

```diff
@@ -737,19 +737,21 @@ class FlutterError extends Error with DiagnosticableTreeMixin implements Asserti
   }
 }
 
-/// Dump the current stack to the console using [debugPrint] and
+/// Dump the stack to the console using [debugPrint] and
 /// [FlutterError.defaultStackFilter].
 ///
-/// The current stack is obtained using [StackTrace.current].
+/// If `stackTrace` is null, dumps the current stack using [StackTrace.current].
```

Comment:
One last nit: this isn't quite a sentence now.  Maybe 
````
If the `stackTrace` parameter is null, the [StackTrace.current] is used to obtain the stack.
````

Reply:
Well, it's a sentence, but maybe it's not in the preferred style and doesn't read as well. =)

I'll change it to your suggestion.